tests/e2e/util.go

```diff
@@ -1055,7 +1055,7 @@ func writeConfigToSecretString(cfg e2eTestConfig) (string, error) {
 }
 
 // Function to create CnsRegisterVolume spec, with given FCD ID and PVC name
-func getCNSRegisterVolumeSpec(ctx context.Context, namespace string, fcdID string, persistentVolumeClaimName string, accessMode v1.PersistentVolumeAccessMode) *cnsregistervolumev1alpha1.CnsRegisterVolume {
+func getCNSRegisterVolumeSpec(ctx context.Context, namespace string, fcdID string, vmdkPath string, persistentVolumeClaimName string, accessMode v1.PersistentVolumeAccessMode) *cnsregistervolumev1alpha1.CnsRegisterVolume {
 	var (
 		cnsRegisterVolume *cnsregistervolumev1alpha1.CnsRegisterVolume
```

Comment:
cnsRegisterVolume ->  cnsRegisterVolumeSpec

Reply:
it is a variable of "CnsRegisterVolume" struct , hence i named the same